README.md

```diff
@@ -20,8 +20,6 @@ https://fishfight.github.io/punchy/demo/
 | Throw Item        | C                               | Period                   | West             |
```

Comment:
Maybe we should totally remove the controller input binding list here?

Users can now view/change the bindings in the game, and it's always easy to forget to update the readme when we add/remove a new move or we change the bindings.

Reply:
I think it would be fine to remove now that its possible to see in the controller settings menu.